src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -127,8 +33,12 @@ def _select_orchestrator(self):
 
         return o
 
-
-    def _list_devices(self, cmd):
+    @CLIReadCommand('orchestrator device ls',
+                    "name=host,type=CephString,req=false "
```

Comment:
```suggestion
                    "name=host,type=CephString,n=N,req=false "
```
With that you can get a list of hosts for free.

Reply:
added